### ISSUE TITLE
Colorimeter Correction Information error

### ISSUE THREAD

**Issue by eoyilmaz** [eoyilmaz/displaycal-py3#132]
Error when trying to get Colorimeter Correction Information.

```shell
DisplayCAL: Uninhibited org.gnome.SessionManager
NoneType: None
┌──────────────────────────────────────────────────────────────────────────────┐
│ Traceback (most recent call last):                                           │
│   File                                                                       │
│ "/home/eoyilmaz/temp/test_displaycal/lib64/python3.10/site-packages/DisplayCAL/display_cal.py", │
│ line 12392, in colorimeter_correction_info_handler                           │
│     plotwindow = CCXXPlot(self, cgats, self.worker)                          │
│   File                                                                       │
│ "/home/eoyilmaz/temp/test_displaycal/lib64/python3.10/site-packages/DisplayCAL/wxCCXXPlot.py", │
│ line 417, in __init__                                                        │
│     bg.Sizer.Add(label, flag=wx.ALIGN_CENTER | wx.ALL & ~wx.TOP, border=16 * │
│ scale)                                                                       │
│   File                                                                       │
│ "/home/eoyilmaz/temp/test_displaycal/lib64/python3.10/site-packages/DisplayCAL/wxfixes.py", │
│ line 702, in Add                                                             │
│     return wx._Sizer.Add(self, *args, **kwargs)                              │
│ TypeError: Sizer.Add(): arguments did not match any overloaded call:         │
│   overload 1: 'flag' is not a valid keyword argument                         │
│   overload 2: argument 'border' has unexpected type 'float'                  │
│   overload 3: argument 2 has unexpected type 'StaticText'                    │
│   overload 4: argument 2 has unexpected type 'StaticText'                    │
│   overload 5: argument 2 has unexpected type 'StaticText'                    │
│   overload 6: argument 2 has unexpected type 'StaticText'                    │
│   overload 7: argument 2 has unexpected type 'StaticText'                    │
│   overload 8: argument 2 has unexpected type 'StaticText'                    │
│   overload 9: argument 2 has unexpected type 'StaticText'                    │
└──────────────────────────────────────────────────────────────────────────────┘
Executing BaseApp.OnExit()
Running application exit handlers
Waiting for thread ScriptingHost.ConnectionHandler to exit
ScriptingHost.ConnectionHandler exited
Exiting displaycal
Ran application exit handlers
```

**Comment by eoyilmaz** [comment]
This is on Fedora 36 with Python 3.10 by the way.

**Comment by eoyilmaz** [comment]
Fixed in a49b302
